### PR TITLE
Add badge component and test

### PR DESCRIPTION
**Pull request by jiggsaw85**
**Jira ticket:**[https://reblaze.atlassian.net/jira/software/projects/V5GA/boards/39?selectedIssue=V5GA-6]( https://reblaze.atlassian.net/jira/software/projects/V5GA/boards/39?selectedIssue=V5GA-6)

### CODE REVIEW

File: src/components/messages/BadgeMessage.vue [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  disabled: false,
+})
+
+</script>
+
+<style lang="scss">
+@import 'src/assets/styles/colors';
+
+.badge-holder {
+  margin: 10px;
```
Comment (Aviv-Galmidi) [inline src/components/messages/BadgeMessage.vue:36]: Why do we need this margin? I can't seem to find it in the figma design

File: src/components/messages/__tests__/BadgeMessage.spec.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import {beforeEach, test, expect, describe} from '@jest/globals'
+import {mount} from '@vue/test-utils'
+import BadgeMessage from '@/components/messages/BadgeMessage.vue'
+
+describe('BadgeMessage.vue', () => {
+  let wrapper: any
```
Comment (Aviv-Galmidi) [inline src/components/messages/__tests__/BadgeMessage.spec.ts:6]: Type should be `VueWrapper`

```suggestion
  let wrapper: VueWrapper
```

**Review by Aviv-Galmidi — CHANGES_REQUESTED** [review]
Please add storybook story preview as well

**Comment by Aviv-Galmidi** [comment]
Please keep the component directory flat `src/components/BadgeMessage.vue` rather than `src/components/messages/BadgeMessage.vue`

File: src/components/messages/BadgeMessage.vue (hunk 1 above)
Comment (jiggsaw85) [reply src/components/messages/BadgeMessage.vue:36]: We do not need it. I put it to make a small distance from badge by default when badge component is loaded.
We can remove it.

File: src/components/messages/__tests__/BadgeMessage.spec.ts (hunk 2 above)
Comment (jiggsaw85) [reply src/components/messages/__tests__/BadgeMessage.spec.ts]: @Aviv-Galmidi All fixed and pushed.

**Review by Aviv-Galmidi — DISMISSED** [review]

**Review by Aviv-Galmidi — COMMENTED** [review]
We're still missing a story preview for storybook
https://storybook.js.org/docs/vue/writing-stories/introduction#how-to-write-stories

It's a short file with example of usage of the component, which will add it to our storybook (component preview library)

**Pull request merged**